src/core/polymer.cpp

```diff
@@ -67,6 +66,23 @@ template <class RNG> static Utils::Vector3d random_unit_vector(RNG &rng) {
   return v;
 }
 
+/** Calculate minimal distance to point.
+ *  Note: Particle handles may be invalidated!
+ *  @param partCfg particle selection
+ *  @param pos  point from which to check for close neighbors
```

Comment:
Double space after "pos"; although, may just be the default indent before the parameter description... caught my eye, though

Reply:
In doxygen comments, the amount of whitespace between a parameter name and its description is arbitrary. In the past I tried to make it more consistent in the codebase by properly aligning the descriptions, but it's time-consuming. And as soon as a new parameter is introduced whose name is a bit too long, or a `@param[in,out]` is changed to a `@param[in]`, the entire list of `@param` needs to be reformatted, possibly with extra line breaks (which can't be masked with `git diff --color-words` because newlines start with an asterisk). Not really worth the effort, and on some occasions it made the resolution of merge conflicts more difficult.